eclass/coreos-sec-policy.eclass

```diff
@@ -0,0 +1,31 @@
+# Copyright 2014 CoreOS, Inc.
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: coreos-sec-policy.eclass
+# @BLURB: Container Linux sec-policy ebuild routines.
+
+
+# When updating the sec-policy ebuilds BASEPOL needs to be updated to match
+# the ${PVR} version string of the upstream selinux policy packages.
+# BASEPOL must corespond to a published gentoo policy patchbundle.
+export BASEPOL="2.20170805-r3"
+
+# Setting these variables allows the sec-policy makefiles to work with
+# Container Linux.
+export BINDIR="/usr/bin"
+export SHAREDIR="${EROOT:-/}usr/share/selinux"
+
+# For build debugging.
+export QUIET="n"
+
+coreos-sec-policy_pkg_postinst() {
+	debug-print-function ${FUNCNAME} "$@"
+
+	# For board builds Container Linux installs policy modules to the OS
+	# image in the SDK build_image script.
+	[[ "${CBUILD}" == "${CHOST}" ]] || return
+
+	selinux-policy-2_pkg_postinst
```

Comment:
I would have liked to do that, but that caused problems.   This new `coreos-sec-policy.eclass` has common things intended for use by all the sec-policy packages.  selinux-policy-2.eclass is intended for use by the policy `add-on` packages like `selinux-virt` and `selinux-unconfined`, and not for the base policy package.  Errors occur if `selinux-policy-2` is inherit in selinux-base.

One way to avoid having to inherit both is to have a conditional on ${PN} inside `coreos-sec-policy.eclass` to inherit `selinux-policy-2`.  Would you prefer that?

Reply:
It seems the above is no longer true.  It may have been a condition of older packages.  Things seem to work OK with a simple `inherit selinux-policy-2` in coreos-sec-policy.eclass.

Reply:
I just did a fresh sdk build and it turns out that the problem is still there.  `selinux-policy-2.eclass` has a `DEPEND` on `selinux-base-policy`, and so creates a circular dependency if either `selinux-base` or `selinux-base-policy` inherit `selinux-policy-2.eclass`.  I added a conditional to `coreos-sec-policy.eclass` to fix this.